### PR TITLE
Rel 4.16.0.1

### PR DESCRIPTION
**Pull request by IgorBagnucki**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -553,7 +556,13 @@ redundant code which was similar for APU2/3/5 boards.
 - turn off D4 and D5 leds on boot
 - enable power on after power failure
 
-[Unreleased]: https://github.com/pcengines/coreboot/compare/v4.14.0.4...develop
+[Unreleased]: https://github.com/pcengines/coreboot/compare/v4.16.0.1...develop
```
Comment (miczyg1) [inline CHANGELOG.md:559]: Something is wrong here. Why do we add so much links?

Comment (IgorBagnucki) [reply CHANGELOG.md:559]: The links were missing in previous releases. For example, on the master branch, v4.14.0.4 has the proper link, but v4.14.0.5 has no link and is surrounded by square brackets. https://github.com/pcengines/coreboot/blob/release/CHANGELOG.md
This change should fix the missing links.

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,14 @@ official [coreboot repository](https://review.coreboot.org/cgit/coreboot.git)
 Please use [pce-fw-builder](https://github.com/pcengines/pce-fw-builder)
 
 ## [Unreleased]
+
+## [v4.16.0.1] - 2022-02-11
+### Changed
+- Rebased with official coreboot repository commit b4ba289f
```
Comment (miczyg1) [inline CHANGELOG.md:18]: we changed the markers around the debug prints in coreboot, this also qualifies as a change

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (IgorBagnucki) [reply CHANGELOG.md:18]: I have added an information about loglevel prefixes and ANSI escapes being disabled.

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**